clang/test/Driver/ftarget-compile-fast-old-model.cpp

```diff
@@ -0,0 +1,20 @@
+// Test -ftarget-compile-fast behaviors
```

Comment:
Some of the comments are updated with // Old offloading model only
Can be made consistent by updating all comments as necessary.

Reply:
The files that are commented with 'Old offload model only' are tests that already exist that were not converted for the new model as those behaviors being tested do not make sense for the new model.